fontforge/featurefile.c

```diff
@@ -7304,17 +7329,29 @@ static void CopySplineFontGroupsForFeatureFile(SplineFont *sf, struct parseState
   }
 }
 
+static bool fea_isInUFO(char *filename) {
+    // Note: Per standard, ^ and $ are implicitly placed at tail and end, so
+    // these aren't defined and can't be used.
+    static xmlChar *regex_pattern = ".*ufo[23]?[/\\\\]features.fea";
```

Comment:
mark const

You could also probably just replace the whole thing with something like

```
return g_regex_match_simple("^.*ufo[23]?[/\\\\]features.fea$", filename, 0, 0);
```